schemas/groups/resources.json

```diff
@@ -202,14 +202,14 @@
             },
             "region": {
               "type": "string",
-              "description": "Region related to note. A pointer to a region UUID. Alternative to position or geohash"
+              "description": "Region related to note. A pointer to a region UUID. Can be defined in additon to position or geohash"
             },
             "position": {
-              "description": "Position related to note. Alternative to region or geohash",
+              "description": "Position related to note. Alternative to geohash",
               "$ref": "../definitions.json#/definitions/position"
             },
             "geohash": {
-              "description": "Position related to note. Alternative to region or position",
+              "description": "Position related to note. Alternative to position",
```

Comment:
As region (arbitrary geographic bounds), geohash (covering some area, small or large) and a precise location are different things I would be happy to change the wording to allow all of them.

Reply:
Yes, allow all of them. They are basically all different ways of filtering by location, having all of them is a bonus. A server might even auto-calculate the others, eg a geohash for a location, or enclosing a region.

Reply:
Updated to indicate all are allowed.